README.md

```diff
@@ -9,20 +13,44 @@ A modern approach to Neovim plugin management.
 
 ## :star2: Features
 
-- `Cargo`-like `rocks.toml` file for declaring all your plugins
-- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead)
-- Automatic dependency and build script management
+- `Cargo`-like `rocks.toml` file for declaring all your plugins.
+- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead).
+- Automatic dependency and build script management.
 - True semver versioning!
-- (WIP) Automatic running of test suites
+- (WIP) Automatic running of test suites.
+
+## :pencil: Requirements
+
+- An up-to-date `Neovim` nightly (>= 0.10) installation.
+- A C/C++ compiler of your choice (must be compatible with `C++17`!) + [`CMake`](https://cmake.org/).
+- [`rust`](https://www.rust-lang.org/) toolchain (recommended latest stable version).
+- [`luarocks`](https://luarocks.org/) installed on your system and accessible in your shell.
```

Comment:
thought: maybe we should consider [hosting our dependencies as binary rocks](https://github.com/luarocks/luarocks/wiki/Hosting-binary-rocks).

Reply:
I didn't know something like this was possible. We should definitely do this, it would reduce dependencies and make the process much faster, I'll take a look at it :p